controllers/touristAttractionController.js

```diff
@@ -8,3 +9,56 @@ exports.getAllAttractions = async (req, res) => {
     res.status(500).json({ message: error.message });
   }
 };
+
+//METODO POST
+exports.createAttractions = async (req, res) => {
+  try {
+    const createAttraction = new TouristAttraction({
+      attractionId: req.body.id,
+      name: req.body.name,
+      description: req.body.description,
+      location: req.body.location,
+      image: req.body.image
+    })
+    const newAttraction = await createAttraction.save();
+    res.status(201).json(newAttraction);
+  } catch (error) {
+    res.status(400).json("No se pudo crear la atraccion.")
+  }
+}
+
+//METODO PUT
+exports.updateAttractions = async (req, res) =>{
+  try {
+    const updateAttraction = await TouristAttraction.findByIdAndUpdate(req.params.id, req.body, {
```

Comment:
we can use `updatedAttraction` instead of `updateAttraction` to reflect that updatedAttraction contain the information returned by mongodb related to the attraction that was updated